embulk-test/src/main/java/org/embulk/test/TestingBulkLoader.java

```diff
@@ -33,13 +44,62 @@
                     public void configure(Binder binder)
                     {
                         binder.bind(BulkLoader.class).to(TestingBulkLoader.class);
+                        registerPluginTo(binder, InputPlugin.class, "preview_result", PreviewResultInputPlugin.class);
                     }
                 };
                 return ImmutableList.of(Modules.override(modules).with(ImmutableList.of(override)));
             }
         };
     }
 
+    /**
+     * This plugin is used for TestingEmbulk.InputBuilder.preview().
+     */
+    static final class PreviewResultInputPlugin
```

Comment:
Can this class be `private`, or if used from outside, can we move this class `public` out of `TestingBulkLoader`?